### ISSUE TITLE
Crash to Desktop at beginning of loading

### ISSUE THREAD

**Issue by EamesCa** [Facepunch/garrysmod-issues#2252]
Hey guys, me and some friends have been trying to get into Gmod for the past week. however every time I start the game it goes to a blue screen and then proceeds to crash. No warning or error message.

I've tried the common fixes mentioned (verify game cache, deleted any source games and ClientRegistry.blob, running hl2.exe as admin or in compatibility mode) I also do not have any mods or workshop items.

So basically I'm out of ideas, any help is appreciated, thank you! 
Since I can't attach the dump files directly, I'm hoping it's okay if I share it through Dropbox. 
https://www.dropbox.com/sh/esfbbxvtdj0pa76/AADcY7bcg2z83Vara3-sC55ka?dl=0&s=sl 

(If that's not okay please let me know how I can share the crash reports)

**Comment by robotboy655** [comment]
You are crashing in Awesomium, can't tell you more or provide any help.

**Comment by shadow7483147** [comment]
I suspect I'm crashing from this as well. Just yesterday Garry's Mod had run perfectly fine and now it's doing this exact crash, loads the blue loading screen without a progress bar and crashes.

I'm not seeing a relevant mdmp for it, all the ones I have are from previous months. I just know as much it's crashing as "hl2.exe has stopped working"

**Comment by robotboy655** [comment]
Try switching to http://wiki.garrysmod.com/page/Dev_Branch and see if that fixes it.

**Comment by shadow7483147** [comment]
Crash persists on reinstall (cleared garrysmod folder), as does with the dev branch.
still no minidumps, and the awesomium log doesn't appear to change.

Think it was a windows update that caused it? Had to install updates when i restarted to try to fix this the first time it happened, and for some reason all system sounds were making no noise until i restarted.

**Comment by robotboy655** [comment]
Try validating GMod, TF2, CS:S, etc.

**Comment by shadow7483147** [comment]
1 fil got reacquired on TF2,GMod launches now

**Comment by EamesCa** [comment]
I haven't had that kind of luck sadly, reinstalling all the source games didn't work nor did just having gmod installed

**Comment by robotboy655** [comment]
@EamesCa, Your crash is different, you are crashing in Awesomium.dll, a third-party module we have no access to.

If you would attach a .mdmp files from crashing on the Dev-beta, that might be helpful.

**Comment by robotboy655** [comment]
I'd really love to know what file in TF2 causes the crash problem @shadow7483147 experienced.

**Comment by shadow7483147** [comment]
Found it
tf2_misc_000.vpk was the only file modified when I verified

**Comment by EamesCa** [comment]
@robotboy655 Ah okay gotcha, thanks for the right direction. I'll work on getting that for you when I return home.

**Comment by robotboy655** [comment]
@shadow7483147 Alright thanks. I guess GMod simply fails to read the corrupted file then. If it ever happens again, please upload the file here so I can download it and replicate the problem on my end.
